snmp/pom.xml

```diff
@@ -155,13 +155,10 @@
 			<!-- Chronicle -->
 			<dependency>
 				<groupId>net.openhft</groupId>
-				<artifactId>chronicle-queue</artifactId>
-				<version>${chronicle.queue.version}</version>
-			</dependency>
-			<dependency>
-				<groupId>net.openhft</groupId>
-				<artifactId>chronicle-map</artifactId>
-				<version>${chronicle.map.version}</version>
+				<artifactId>chronicle-bom</artifactId>
+				<type>pom</type>
+				<scope>import</scope>
+				<version>2.22.132</version>
```

Comment:
- [ ] With the introduction of these changes and dependency on chronicle-bom, I guess we don't need the two constants chronicle.map.version and chronicle.queue.version. Can these be removed as they are still pointing to old versions and is also misleading?
- [ ] Also with this change, aren't we pulling and bundling many more chronicle libraries which are not used by the agent? Earlier SNMP agent depended on only two chronicle libraries chronicle-queue and chronicle-map.

Reply:
BOMs in general when used in dependencyManagement section are used to manage dependencies versions. It does not lead to dependency import, it only makes sure a valid versions (compatible with each other) are used in application

Reply:
Got it, thanks Kamil.